cli/bin/monkey-patch-oclif.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Oclif mostly supports ESM, but it doesn't properly support commands
+ * in "index.js" files. It always thinks they are not ESM files.
+ *
+ * This file monkey-patches Oclif with the code
+ * from this PR https://github.com/oclif/core/pull/417 to support them.
+ * The PR has been merged so in the next release we should be able to remove
+ * this monkey patching.
+ */
```

Comment:
Is this still needed after https://github.com/oclif/core/pull/417 was merged?

Reply:
I see there's a release with that change https://github.com/oclif/core/releases/tag/v1.8.2 Last time I checked there was no release yet. I'll remove the monkey-patching